Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -211,7 +222,12 @@ private async UniTaskVoid SearchAndSetEmojiSuggestionsAsync(string value, Cancel
 
         private void CreateChatEntry(ChatMessage chatMessage)
         {
-            world.Create(new ChatBubbleComponent(chatMessage.Message, chatMessage.Sender, chatMessage.WalletAddress));
+            if (chatMessage.SentByOwnUser == false)
+            {
+                var entity = entityParticipantTable.Entity(chatMessage.WalletAddress);
+                world.Add(entity, new ChatBubbleComponent(chatMessage.Message, chatMessage.Sender, chatMessage.WalletAddress));
```

Comment:
I think it's quite dangerous to assume that the component was already processed and deleted: what if for some reason we receive several messages due to the delay?! There will be an exception
Let's change it to Add or Set